src/Display/Modal/Modal.tsx

```diff
@@ -81,6 +81,10 @@ const Modal = (props: Props) => {
       // On modal close
       document.body.removeAttribute('style');
     }
+
+    return () => {
+      document.body.removeAttribute('style');
```

Comment:
According to your commit message, I think we should add `if (mountingState === MountingState.UNMOUNTED)` here.

Hmm, but why, shouldn't line 82 do the same thing?

Reply:
hmm, in our feature, we wrap Modal in a Component, the Modal is used to delete the Component. When the component is unmounted, the Modal's css transition is not finished yet, but it is being unmounted together with the Component.

Because of this, I want to close this PR, I think it's better to let this Modal component have minimum logic.